securebanking-openbanking-uk-rs-simulator/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/obie/event/v3_1_3/callbackurl/CallbackUrlsApiController.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.securebanking.openbanking.uk.rs.api.obie.event.v3_1_3.callbackurl;
+
+import com.forgerock.securebanking.openbanking.uk.rs.persistence.repository.events.CallbackUrlsRepository;
+import org.springframework.stereotype.Controller;
+
+@Controller("CallbackUrlsApiV3.1.3")
+public class CallbackUrlsApiController extends com.forgerock.securebanking.openbanking.uk.rs.api.obie.event.v3_1_2.callbackurl.CallbackUrlsApiController implements CallbackUrlsApi {
+
+    public CallbackUrlsApiController(CallbackUrlsRepository callbackUrlsRepository) {
+        super(callbackUrlsRepository);
+    }
```

Comment:
Only one constructor is required now (compared to https://github.com/OpenBankingToolkit/openbanking-aspsp/blob/master/forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/event/v3_1_3/callbackurl/CallbackUrlsApiController.java)